gopages/internal/generate/template.go

```diff
@@ -138,3 +139,21 @@ func nodeHTML(original nodeHTMLFunc, baseURL string, modulePackage string) nodeH
 		return replacer.Replace(original(info, node, linkify))
 	}
 }
+
+type commentHTMLFunc = func(info *godoc.PageInfo, comment string) string
+
+func commentToHTML(original commentHTMLFunc, baseURL string, modulePackage string) commentHTMLFunc {
+	var (
```

Comment:
Looks like the body of this function is nearly identical to `nodeHTML`. Let's deduplicate a little. Maybe something like this?

```go
func rewriteLinks(rawHTML, baseURL, modulePackage string) string { ... }

func nodeHTML(original nodeHTMLFunc, baseURL, modulePackage string) nodeHTMLFunc {
	return func(info *godoc.PageInfo, node interface{}, linkify bool) string {
		rawHTML := original(info, node, linkify)
		return rewriteLinks(rawHTML, baseURL, modulePackage)
	}
}

func commentToHTML(original commentHTMLFunc, baseURL string, modulePackage string) commentHTMLFunc {
	return func(info *godoc.PageInfo, comment string) string {
		rawHTML := original(info, comment)
		return rewriteLinks(rawHTML, baseURL, modulePackage)
	}
}
```

Reply:
Done